### PR TITLE
Adding QuoteLengthLimit option for Feature request: #963

### PR DESCRIPTION
**Pull request by Humorhenker**
FR #963 asked for an option to limit the max quoted message length.
QuoteLengthLimit option added to limit max. quoted message length.
If QuoteLengthLimit = 0 the whole message will be quoted.

### CODE REVIEW

**Comment by Humorhenker** [comment]
Travis complains about one simple readability improvement.
Am I supposed to create a new PR?

File: bridge/xmpp/xmpp.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/42wim/matterbridge/bridge/config"
 	"github.com/42wim/matterbridge/bridge/helper"
 	"github.com/jpillora/backoff"
-	"github.com/matterbridge/go-xmpp"
+	"github.com/humorhenker/go-xmpp"
```
Comment (42wim) [inline bridge/xmpp/xmpp.go]: It should keep `go-xmpp` from matterbridge

File: bridge/xmpp/xmpp.go [hunk 2]
```diff
@@ -331,7 +331,7 @@ func (b *Bxmpp) skipMessage(message xmpp.Chat) bool {
 	}
 
 	// skip empty messages
-	if message.Text == "" {
+	if message.Text == "" &&  message.Chatstate == "" {
```
Comment (42wim) [inline bridge/xmpp/xmpp.go]: If you want to change this, please open a new PR for this change (it's not related to the QuoteLengthLimit feature)

**Comment by 42wim** [comment]
Thanks for the PR!

I've added some comments to your code.

> Travis complains about one simple readability improvement.
Am I supposed to create a new PR?

No, you can just push commits (that will fix the issues travis complains about) to your branch (master in your case) 

Travis will rebuild this PR with every commit you push.

**Comment by Humorhenker** [comment]
Updated now.
I am still such a git noob. Did not realized that the pr will update with my changes on the fork. Did not mean to include the other changes.

**Comment by 42wim** [comment]
Thanks! No worries, your git-fu will grow :)

**Pull request merged**